projects/canopy/src/styles/mixins.scss

```diff
@@ -31,6 +31,12 @@
   } @else {
     font-weight: var(--text-fs-#{$size}-weight);
   }
+
+  @if $type == 'expressive' {
+    font-family: var(--font-family-#{$type});
+  } @else if $type == 'productive' {
```

Comment:
Just wondering if `productive` should be the default value?
Maybe actually it could work only with this bit:
```
@if $type == 'expressive' {    font-family: var(--font-family-#{$type});  }
```
since the font-family by default it's set to Roboto? Would that work?

Reply:
Well, that was my first thought, but I was thinking about the possibility where some other font would be used in the future. This way we would have an option to set Roboto with the productive parameter. Anyway, I would rather go toward fully generic solution where we would take the type parameter and provide it to the `var(--font-family-#{$type}` and set proper font family.

If you think it's better, I will remove productive parameter part. :)

Reply:
I agree with @elenagarrone, I don't think we need `if $type == 'productive'`, because it's the default font anyway which isset by the CSS var `--font-family`. So I think the logic could look like this:

```
@if $type == 'expressive' {    
  font-family: var(--font-family-#{$type});  
} @else {
  font-family: var(--font-family);
}
```

Future we could add more conditions if we add more fonts, and/or if the default font changed too.

Reply:
In this case we don't need _@else_ part since it will set the **Roboto** font which is already the default font. It will set **Lyon** font only if we provide _$type_ parameter as expressive and will not set font-family property for this mixin if we don't provide this value for the parameter (null or other value will skip this case).